content/docs/overview/tracers.md

```diff
@@ -1,3 +1,51 @@
 ---
 title: Tracers
 ---
+
+## Introduction
+
+OpenTracing provides an open, vendor-neutral standard API for consistent distributed tracing of request across processes, from web and mobile client platforms to the storage and custom backends at the bottom of application. This means that instrumentation would remain the same irrespective of the tracer system being used by the developer. In order to instrument the web application using OpenTracing specification, a compatible OpenTracing tracer must be deployed and listening for incoming span requests. A list of the all the supported tracers is available [here](./../../supported-tracers).
+
+### Tracer Interface
+
+The Tracer interface creates Spans and understands how to Inject (serialize) and Extract (deserialize) them across process boundaries. It has the following capabilities:
+
+- Start a New Span
+- Inject a SpanContect into a Carrier
+- Extract a SpanContect from a Carrier
+
+Each of these will be discussed in more detail below. For implementation purposes, check out the specific language guide.
+
+### Setting up Tracer
+
+The first step in instrumenting your application with OT specification is setting up the tracer configuration and registering the tracer with GlobalTracer using the `register` method. This step basically indicates the tracer’s endpoint and the service that will send data to the server. The GlobalTracer forwards all the methods to the tracer that has been configured.
+The `register()` method can only be called once by the developer in every application during the initialization of a tracer. In case no Global Tracer is registered, the default `NoopTracer` will be used.
+
+### Starting a new Trace
+Finally, a new trace can be started by creating a new span. A span must specify the operation name. Operation names are used for providing semantic meaning to the trace. It is important to recognize whether we need to start a new trace or connect with an already-active trace. If there's an active trace, you'll start a span with the active span as its parent. Otherwise, the span you start will have no parent.OpenTracing supports two kinds of direct causal relationships:
+
+1. **ChildOf**: The parent span is dependent on the child span for its execution.
+2. **FollowsFrom**: This relationship is used to model asynchronous executions. The parent span is fully independent of the child span.
+
+
+### Accessing the Active Span
+The GlobalTracer can be used to access the currently ActiveSpan and log events. Further, ActiveSpans can be accessed through a scopeManager in some languages. Refer to the specific language guide for more implementation details.
```

Comment:
I would prefer to have the instructions to refer only to the "core" API, without the `GlobalTracer`. Then, later on, we could have a chapter on the `GlobalTracer`, showing how it can make life easier. As far as I remember, the `GlobalTracer` is not part of the specification and is not available for all languages.